src/tracy-modification.c

```diff
@@ -116,6 +118,40 @@ int tracy_inject_syscall_pre_start(struct tracy_child *child, long syscall_numbe
     child->inj.pre = 1;
     child->inj.syscall_num = syscall_number;
 
+#ifdef __arm__
+    if(child->tracy->opt & TRACY_WORKAROUND_ARM_7475_1) {
+        /* change call to getpid */
+        tracy_modify_syscall_args(child, get_syscall_number_abi("getpid", child->event.abi), &child->inj.reg);
```

Comment:
oh wow. thx for this I found a makefile bug thx to this. Only my last line with cflags was used